imports/ui/views/users/professionalExperience/ProfessionalExperienceContent.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import ProfessionalExperienceLogo from './ProfessionalExperienceLogo.jsx'
+import ProfessionalExperienceHeader from './ProfessionalExperienceHeader'
+import ProfessionalExperienceDescription from './ProfessionalExperienceDescription'
+
+class ProfessionalExperienceContent extends Component {
+  render() {
+
+    let info = this.props.info
+
+    return (
+      <div style={contentStyle}>
+        <div style={topStyle} >
+          <ProfessionalExperienceLogo logoUrl={info.logoUrl} />
```

Comment:
Should we move the Logo image component into the components folder since we currently have essentially the same logo component being used in both ProfessionalExperience and VolunteerExperience?